shell/apps/defaults.json

```diff
@@ -11,111 +12,127 @@
             "shortcuts": [
                 {
                     "name": {
+                        "ca_ES": "Xarxa",
                         "en_GB": "Network",
                         "fr_FR": "Réseau"
                     },
                     "url": "gshell://apps/settings/index.html?shortcut=network"
                 },
                 {
                     "name": {
+                        "ca_ES": "Xarxes sense fil",
```

Comment:
Should this not be "Xarxes Wi-Fi"? Just for consistency with the translation used in the Settings app